protocols/kad/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.44.3 - unreleased
```

Comment:
```suggestion
## 0.44.3
```

Adding `- unreleased` is the way to go. Thank you. Though in this case I will cut a release right away.